gradle/forge.gradle

```diff
@@ -36,7 +39,10 @@ dependencies {
 
     // Project lombok
     provided "org.projectlombok:lombok:1.16.8"
+
     testCompile "junit:junit:4.12"
+    testCompile "com.jjtparadox.barometer:barometer:0.0.3-1.12"
```

Comment:
This likely needs to be `com.jjtparadox.barometer:Barometer:0.0.3-1.12` (Note the capital B)

Reply:
Oops... I think it was picking up the lowercase version from my mavenLocal repository...